### PR TITLE
feat: First draft of website

### PR DESCRIPTION
**Pull request by frgfm**
This PR adds basic content to the webpage with several sections:
- Header with title & short description
- "Who are we?" section : few words about the association and our core conviction
- "How does it work" section : short description of the system 
- "How to set it up" section : invitation to contact us --> that part should be edited in a whole PR i guess
- "Our sponsors" section : Atraksis + D4G (hopefully we can add the ministry or some departments soon)
- "Contact form" section : to send emails

The jekyll server was run locally and yields the following appearence:
Pyronear - Détection précoce de feux de forêt

_Note: I couldn't manage to get the Twitter button to work in the footer, if someone got an idea :man_shrugging: _

### CODE REVIEW

**Comment by fe51** [comment]
Hi frgfm, thanks for the PR !
 
That is a really good and coherent start to start iterations.
 
So far, I have not tested yet and just get an orverview thanks to the code and the screenshot. 
 
About Atraksis sponsor,
I have the feeling that this suggests that we only interact with firefigters through them.
 
Besides, should we also add (if we manage to), linkedin button as well ?

**Comment by fe51** [comment]
Moreover, it might be nice to add a contributing section, with a link to Pyronear's github and suggestion to contact us for contact suggestion for other areas of expertise.

**Comment by frgfm** [comment]
@fe51  Good idea, I'll try to fix the Twitter button and add the LinkedIn one!
True about Atraksis, any idea about what we might want to change it to?

Same if you have suggestions about the Contribution sections (perhaps we should do this in another PR?)

**Comment by MateoLostanlen** [comment]
Hi, Thanks for the PR! 

Overall it's great, simple but effective. I have two small remarks:

1. I think the word 'free' should appear in the description paragraph. Here it looks like we're trying to keep costs down but we're still selling something. 

2. I'm not a fan of the logo for image processing.

@fe51 , totally agree for the github. For Atraksis this is not an exhaustive list of our partners but only those who support us. But I understand your remark

**Comment by frgfm** [comment]
Here is the 2nd draft with @fe51 suggestions!
Pyronear - Détection précoce de feux de forêt

@MateoLostanlen are you referring to the "Who are we?" section or the one above it?
Any suggestions of icon to replace the previous one?

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      moreInfo: "En savoir plus"
+      contactUs: "Nous contacter"
+      submit: "Envoyer"
+    header:
+      title: "Détection précoce des départs de feux"
+      subtitle: "Pyronear est un projet open-source au service du bien commun."
+    manifesto:
+      title: "Qui sommes-nous ?"
+      phrase1: "Pyronear est une association loi 1901 au service du bien commun et du logiciel libre."
+      phrase2: "Nous portons la conviction que la protection des forêts doit être accessible à tous et qu'elle nécessite la technologie la plus efficace, transparente et abordable possible."
```
Comment (MateoLostanlen) [inline _config.yml:28]: Maybe we can add that:
 "Nous portons la conviction que la protection des forêts doit être accessible à tous et qu'elle nécessite la technologie la plus efficace, transparente et abordable possible. C’est pourquoi nous souhaitons proposer gratuitement notre solution pour détecter les feux de forêt. "
In order to make clear that our solution is free

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+# Traduction
+t:
+  fr:
+    buttons:
+      moreInfo: "En savoir plus"
+      contactUs: "Nous contacter"
+      submit: "Envoyer"
+    header:
+      title: "Détection précoce des départs de feux"
+      subtitle: "Pyronear est un projet open-source au service du bien commun."
```
Comment (MateoLostanlen) [inline _config.yml:24]: I will remove the dot at the end, I always find strange a dot in a title or subtitle. But it's personal

**Comment by MateoLostanlen** [comment]
About the Image processing logo, this robot looks a bit like a clown. We can take an AI style logo : 

https://pngtree.com/freepng/artificial-intelligence-glyph-vector-icon-set_3753946.html

File: _config.yml (hunk 1 above)
Comment (frgfm) [reply _config.yml]: Done!

File: _config.yml (hunk 2 above)
Comment (frgfm) [reply _config.yml]: Fixed!

**Comment by MateoLostanlen** [comment]
Maybe we can add something like 'don't hesitate to contact us if you want to join the adventure'

**Review by MateoLostanlen — APPROVED** [review]
All good for me 😊

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      subtitle: "Pyronear est un projet open-source au service du bien commun"
+    manifesto:
+      title: "Qui sommes-nous ?"
+      phrase1: "Pyronear est une association loi 1901 au service du bien commun et du logiciel libre."
+      phrase2: "Nous portons la conviction que la protection des forêts doit être accessible à tous et qu'elle nécessite la technologie la plus efficace, transparente et abordable possible. C’est pourquoi notre solution logicielle de détection des feux de forêt est gratuite."
+    steps:
+      title: "Comment ça marche ?"
+      step1:
+        title: "1. Couverture de la zone"
+        content: "Nos détecteurs placés en hauteur surveillent la zone forestière alentour."
```
Comment (Akilditu) [inline _config.yml:33]: Maybe the word 'cameras' instead of 'détecteurs' would make more sense here

**Review by Akilditu — APPROVED** [review]
Hi guys, @frgfm thanks a lot for this PR. 

Everything seems fine for merge !

**Pull request merged**